packages/components/src/pages/ChatSearch.vue

```diff
@@ -250,6 +250,30 @@ export default {
     },
   },
   methods: {
+    async loadThread(threadId: string) {
+      const thread = await this.rpcClient.loadThread(threadId);
+      const chat = this.$refs.vchat;
```

Comment:
Should `chat.threadId` be set here as well?

Reply:
Yeah it seems like it should

Reply:
This is addressed now.